packaging/helm-charts/helm3/strimzi-kafka-operator/templates/023-RoleBinding-strimzi-cluster-operator.yaml

```diff
@@ -1,6 +1,10 @@
-{{- if .Values.rbac.create }}
+{{- if .Values.rbac.create -}}
```

Comment:
Out of curiosity, what does the `-` at the end do here?

Reply:
Notice that we received a few empty lines in our YAML. Why? When the template engine runs, it removes the contents inside of {{ and }}, but it leaves the remaining whitespace exactly as is.

YAML ascribes meaning to whitespace, so managing the whitespace becomes pretty important. Fortunately, Helm templates have a few tools to help.
First, the curly brace syntax of template declarations can be modified with special characters to tell the template engine to chomp whitespace. {{- (with the dash and space added) indicates that whitespace should be chomped left, while -}} means whitespace to the right should be consumed. Be careful! Newlines are whitespace!

If it is the first line in a yaml file, I think it will not impact anything.
If you concern about it, I can remove - at the end.